src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -568,7 +569,7 @@ case class JesBackend(actorSystem: ActorSystem)
       status match {
         case Run.Success(events) if backendCall.runtimeAttributes.failOnStderr && stderrLength.intValue > 0 =>
           // returnCode will be None if it couldn't be downloaded/parsed, which will yield a null in the DB
-          FailedExecutionHandle(new Throwable(s"${log.tag} execution failed: stderr has length $stderrLength"), returnCode.toOption).future
+          FailedExecutionHandle(new Throwable(s"${log.tag} execution failed: stderr has length $stderrLength"), returnCode.toOption map ScriptReturnCode).future
```

Comment:
Oof, those are the sorts of implicits I hate because it can be very non-obvious that a conversion is happening. A normal function is fine too, I just prefer the .toXYZ kinda syntax.

After I wrote this earlier I noticed that besides the two here there were some variants here and there as well - some were already options, some weren't.